dist/index.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Development Harness</title><link rel="stylesheet" href="https://cfpb.github.io/capital-framework/dist/static/css/main.min.css"><script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.5.4/react.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.5.4/react-dom.js"></script><script src="https://cfpb.github.io/capital-framework/dist/static/js/main.min.js"></script></head><body><div id="root"></div><script type="text/javascript" src="ccdb5.main.js"></script></body><script type="text/javascript">ReactDOM.render(React.createElement(ccdb5_ui.App,{},null),document.getElementById("root"))</script></html>
```

Comment:
The idea is that the CCDB5 code in cfgov-refresh would include this repo as a dependency, so it would only use the files in the `dist` directory.